### PR TITLE
Update token setting for delegate kubernetes yaml

### PR DESCRIPTION
**Pull request by xingchi-jin**
Current document is showing token setting for legacy delegate. In ng, we have rolled out immutable delegate.

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md [hunk 1]
```diff
@@ -67,20 +67,28 @@ The Delegate is set up using the **harness-delegate.yaml** you downloaded origin
 
 Edit the **harness-delegate.yaml** you downloaded originally with the new token and then run `kubectl apply -f harness-delegate.yaml` to restart the Delegate pods.
 
-Paste the token in the Delegate `ACCOUNT_SECRET` setting in the `StatefulSet` spec:
+Paste the token in the Delegate `UPGRADER_TOKEN` setting in the `Secret` spec and `DELEGATE_TOKEN` setting setting in the `Secret` spec:
```
Comment (kat-enos) [inline docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md:70]:
```suggestion
Paste the token into delegate `spec` of the `Secret`, in the `UPGRADER_TOKEN` field. Then paste the token into the specification of the `Secret` in the `DELEGATE_TOKEN` field. 
```

**Review by kat-enos — APPROVED** [review]
Hi Xingchi. Thanks for doing this. I think must be what Vishal was talking about. I'm approving it but have edited it and hope that you'll check it to ensure that I understood your. intention. Please commit my changes or make your own and have me look at it again. Thank you.

**Comment by KimberlyFields** [comment]
@xingchi-jin please review Katherine's suggested revisions and commit them if you agree with them or comment with reasons why the revisions are not correct. Thank you.

**Comment by kat-enos** [comment]
@xingchi-jin Hi Xingchi. I happened to notice this PR and it looks like something we need to finish up. A conflict had developed and I resolved it by eliminating the section of YAML you marked to be deleted. If you could read-through and commit my edits, if they are technically correct, we can merge this. Thank you.

**Review by kat-enos — APPROVED** [review]

**Pull request merged**